charts/darts-api/values.yaml

```diff
@@ -94,7 +94,7 @@ function:
   aadIdentityName: darts
   pollingInterval: 60
   minReplicaCount: 0
-  maxReplicaCount: 2
+  maxReplicaCount: 5
```

Comment:
We should probably do this in flux, otherwise we will get 5 replicas on each cluster in each environment?!